data/flexibleForms/index.ts

```diff
@@ -0,0 +1,12 @@
+import allForms from './forms.json';
+import { Form } from './forms.types';
+
+const formData: Form[] = [];
+
+for (let i = 0; i < allForms.length; i++) {
+  const form = allForms[i] as unknown as Form;
+
+  formData.push(form);
+}
+
+export default formData;
```

Comment:
is this the file we should be using to grab all the forms now? i notice that further down one of the json files is being directly imported